src/api/application/application.ts

```diff
@@ -69,7 +69,7 @@ export default class ApplicationModule extends Bare {
      * @tutorial application.createFromManifest
      */
     public createFromManifest(manifestUrl: string): Promise<Application> {
-        return this.wire.sendAction('get-application-manifest', {manifestUrl})
+        return this.wire.sendAction<any, {manifestUrl: string}>('get-application-manifest', {manifestUrl})
```

Comment:
any => Application

Reply:
sendAction doesn't return an Application, but an an Application Manifest

Reply:
no, it should return a wrapped application.